### PR TITLE
Profile public title and misc contribution page improvements

### PR DESCRIPTION
**Pull request by agh1**

### CODE REVIEW

**Comment by agh1** [comment]
Towards #278

File: docs/contributions/online-contributions.md [hunk 1]
```diff
@@ -100,10 +98,10 @@ This is covered in detail in  [Memberships](/membership/online-membership-sign-u
 
 ### Profile tab
 
-If you want to collect information from contributors beyond the essential fields
-required to make a contribution, such as age, interests and skills,
-you can include existing CiviCRM Profiles at the beginning or end of a
-contribution page. You can also create new profiles.
+If you want to collect information such as age, interests and skills that goes
+beyond the essential fields required to make a contribution, you can include
+existing [CiviCRM Profiles](/organising-your-data/profiles) at the beginning or
```
Comment (seamuslee001) [inline docs/contributions/online-contributions.md]: @agh1 can you put a .md on the end of the url as its an internal url

Comment (agh1) [reply docs/contributions/online-contributions.md]: None of the other URLs on the page have it--I thought I was told once *not* to do it.  If the proper style is to include the `.md` I'm happy to fix all the links in the file.

Comment (seancolsen) [reply docs/contributions/online-contributions.md]: I know. I'm sorry this is so confusing! It's explained here: https://docs.civicrm.org/dev/en/latest/documentation/markdown/#internal-url-standards 

The `.md` isn't actually necessary, but adding it helps us avoid broken internal links. For links between guides, the URL should be treated as external and so you should grab the URL you see in the browser, which won't have a `.md`.

**Comment by agh1** [comment]
Okay finished with work on this page.  @seanmadsen thanks for the explanation on the `.md`, I just went through and changed them all on the page.

**Comment by seamuslee001** [comment]
I have reviewed these changes and they make sense. The screenshots are good. Merging

**Pull request merged**